geode-core/src/main/java/org/apache/geode/internal/net/SSLUtil.java

```diff
@@ -36,48 +35,15 @@
 
 public class SSLUtil {
   /**
-   * This is a list of the algorithms that are tried, in order, when "any" is specified. Update
+   * This is a list of the algorithms that are tried, in order, to create an SSLContext. Update
    * this list as new algorithms become available and are supported by Geode. Remove old,
    * no-longer trusted algorithms.
    */
   static final String[] DEFAULT_ALGORITHMS = {"TLSv1.3", "TLSv1.2"};
```

Comment:
I wonder now if this makes more sense as `SUPPORTED_CONTEXTS` or `SUPPORTED_PROTOCOLS` now?

Reply:
Good call. I think `SUPPORTED_CONTEXTS` is best, since my understanding is that these contexts both technically allow support for other protocols.